test/alternator/test_ttl.py

```diff
@@ -294,6 +294,52 @@ def test_ttl_expiration(dynamodb):
         assert 'Item' in table.get_item(Key={'p': p6})
         assert 'Item' in table.get_item(Key={'p': p7})
 
+# test_ttl_expiration above used a table with just a hash key. Because the
+# code to *delete* items in a table which also has a range key is subtly
+# different than the code for just a hash key, we want to test the case of
+# such a table as well. This test is simpler than test_ttl_expiration because
+# all we want to check is that the deletion works - not the expiration time
+# parsing and semantics.
+@pytest.mark.veryslow
+@pytest.mark.xfail(reason="TTL not implemented yet #5060")
+def test_ttl_expiration_with_rangekey(dynamodb):
+    # Note that unlike test_ttl_expiration, this test doesn't have a fixed
+    # duration - it finishes as soon as the item we expect to be expired
+    # is expired.
+    if is_aws(dynamodb):
+        max_duration = 1200
+    else:
+        max_duration = 3
```

Comment:
ditto about the `x if something else y` syntax

Reply:
Did this in the next version I'll send.